test.properties

```diff
@@ -561,6 +561,12 @@
 ## Gradle Plugins
 ##
 
+    gradle.plugins.test.excludes=\
+        gradle-plugins-workspace
+    
+    gradle.plugins.test.excludes.globs[gradle-plugins-workspace]=\
+        *StartDockerContainer*
```

Comment:
@kyle-miho this will also disable the `startDockerContainerWith*` tests. Are they failing as well? If not we should make this glob a bit more specific.